robottelo/test.py

```diff
@@ -132,13 +134,26 @@ class UITestCase(TestCase):
     def setUpClass(cls):  # noqa
         """Make sure that we only read configuration values once."""
         super(UITestCase, cls).setUpClass()
-        cls.katello_user = settings.server.admin_username
-        cls.katello_passwd = settings.server.admin_password
+
+        cls.katello_default_user = settings.server.admin_username
+        cls.katello_default_password = settings.server.admin_password
+
+        cls.katello_user = entities.User(
```

Comment:
Maybe we should rename this to foreman_user, the old katello naming was used when the katello was the main project and now foreman is where the users are defined.